core/src/main/resources/hudson/model/DirectoryBrowserSupport/dir.jelly

```diff
@@ -124,13 +124,13 @@ THE SOFTWARE.
            </table>
            <j:if test="${showSymlinkWarning}">
              <p>
-                <img id="alert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" />
+                <img id="alert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" alt="warning image"/>
                ${%Symlinks are hidden}
              </p>
             </j:if>
            <j:if test="${showTmpDirWarning}">
              <p>
-                <img id="alert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" />
+                <img id="alert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" alt="warning image"/>
```

Comment:
Since it is a temporary directory warning, let's use that as alt text
```suggestion
                <img id="alert" src="${imagesURL}/svgs/warning.svg" width="16" height="16" alt="temporary directory warning"/>
```